### PR TITLE
Create consistency in TEST evaluation section based on instructions

### PR DESCRIPTION
**Pull request by Diolor**
## Description

Expand instructions to define how to write "happy" (pass) cases in v2 tests.

As a consiquence:

1. Align the tests to focus on "fail" cases. Remove redundant "pass clauses".
2. Add the word "case" where missing as per instructions.
3. In evaluation use `!!! note` instead of `**note:**`

---

## AI Tool Disclosure

- [ x] This contribution does not include AI-generated content.

### CODE REVIEW

File: tests-beta/ios/MASVS-AUTH/MASTG-TEST-0268.md [hunk 1]
```diff
@@ -27,8 +27,7 @@ The output should contain a list of locations where relevant APIs are used.
 
 ## Evaluation
 
-The test fails if the app uses `SecAccessControlCreateWithFlags` with the `kSecAccessControlUserPresence` or `kSecAccessControlDevicePasscode` flags for any sensitive data resource that needs protection.
+The test case fails if the app uses `SecAccessControlCreateWithFlags` with the `kSecAccessControlUserPresence` or `kSecAccessControlDevicePasscode` flags for any sensitive data resource that needs protection.
 
-The test passes only if the app uses `SecAccessControlCreateWithFlags` with stricter flags, such as [`kSecAccessControlBiometryAny`](https://developer.apple.com/documentation/security/secaccesscontrolcreateflags/biometryany), [`kSecAccessControlBiometryCurrentSet`](https://developer.apple.com/documentation/security/secaccesscontrolcreateflags/biometrycurrentset) to enforce biometric-only access for any sensitive data resource that needs protection (being `kSecAccessControlBiometryCurrentSet` the one considered the most secure).
```
Comment (Diolor) [inline tests-beta/ios/MASVS-AUTH/MASTG-TEST-0268.md:32]: RFC: This is removed. Anything valuable to move to a different location?

File: tests-beta/ios/MASVS-AUTH/MASTG-TEST-0269.md [hunk 2]
```diff
@@ -23,6 +23,4 @@ The output should contain a list of locations where the `SecAccessControlCreateW
 
 ## Evaluation
 
-The test fails if the app uses `SecAccessControlCreateWithFlags` with the `kSecAccessControlUserPresence` or `kSecAccessControlDevicePasscode` flags for any sensitive data resource that needs protection.
-
-The test passes only if the app uses `SecAccessControlCreateWithFlags` with stricter flags, such as [`kSecAccessControlBiometryAny`](https://developer.apple.com/documentation/security/secaccesscontrolcreateflags/biometryany), [`kSecAccessControlBiometryCurrentSet`](https://developer.apple.com/documentation/security/secaccesscontrolcreateflags/biometrycurrentset) to enforce biometric-only access for any sensitive data resource that needs protection (being `kSecAccessControlBiometryCurrentSet` the one considered the most secure).
```
Comment (Diolor) [inline tests-beta/ios/MASVS-AUTH/MASTG-TEST-0269.md:28]: RFC: This is removed. Anything valuable to move to a different location?

File: tests-beta/android/MASVS-AUTH/MASTG-TEST-0326.md [hunk 3]
```diff
@@ -30,9 +30,7 @@ The output should include a list of locations where the relevant APIs are used.
 
 ## Evaluation
 
-The test fails if the app uses `BiometricPrompt` with authenticators that include `DEVICE_CREDENTIAL` for any sensitive data resource that needs protection.
-
-The test passes if the app uses only `BiometricPrompt` with `BIOMETRIC_STRONG` to enforce biometric-only access for any sensitive data resource that needs protection.
```
Comment (Diolor) [inline tests-beta/android/MASVS-AUTH/MASTG-TEST-0326.md:35]: RFC: This is removed. Anything valuable to move to a different location?

File: tests-beta/android/MASVS-AUTH/MASTG-TEST-0327.md [hunk 4]
```diff
@@ -28,12 +28,7 @@ The output should include a list of locations where the relevant APIs are used.
 
 ## Evaluation
 
-The test fails for each sensitive operation worth protecting if:
+The test case fails for each sensitive operation worth protecting if:
```
Comment (Diolor) [inline tests-beta/android/MASVS-AUTH/MASTG-TEST-0327.md]: wondering if that's "and" or "or". It's not explicit

File: .github/instructions/mastg-test.instructions.md [hunk 5]
```diff
@@ -253,4 +253,18 @@ Example:
...
+
+In that case, it MUST start with "The test case passes if ..." and a fail explanation must not be added.
+
+Example:
+
+```md
+## Evaluation
+
+The test passes if all security-sensitive inputs are protected against keyboard caching.
+```
```
Comment (cpholguera) [inline .github/instructions/mastg-test.instructions.md]:
````suggestion
In that case, it MUST start with "The test case passes if ..." and must be added after the fail explanation.
````

File: tests-beta/android/MASVS-AUTH/MASTG-TEST-0327.md (hunk 4 above)
Comment (cpholguera) [reply tests-beta/android/MASVS-AUTH/MASTG-TEST-0327.md]:
```suggestion
The test case fails for each sensitive operation worth protecting if all of the following applies:
```

File: tests-beta/android/MASVS-NETWORK/MASTG-TEST-0234.md [hunk 6]
```diff
@@ -30,4 +30,5 @@ The output should contain a list of locations where `SSLSocket` and `HostnameVer
 
 The test case fails if the app uses `SSLSocket` without a `HostnameVerifier`.
 
-**Note:** If a `HostnameVerifier` is present, ensure it's not implemented in an unsafe manner. See @MASTG-TEST-0283 for guidance.
+!!! note
+ If a `HostnameVerifier` is present, ensure it's not implemented in an unsafe manner. See @MASTG-TEST-0283 for guidance.
```
Comment (cpholguera) [inline tests-beta/android/MASVS-NETWORK/MASTG-TEST-0234.md]:
```suggestion
    If a `HostnameVerifier` is present, ensure it's not implemented in an unsafe manner. See @MASTG-TEST-0283 for guidance.
```

File: tests-beta/android/MASVS-NETWORK/MASTG-TEST-0235.md
Comment (cpholguera) [inline tests-beta/android/MASVS-NETWORK/MASTG-TEST-0235.md]:
````suggestion
    The test doesn't fail if the AndroidManifest sets `usesCleartextTraffic` to `true` and there's a NSC, even if it only has an empty `<network-security-config>` element. For example:
    
    ```xml
    <?xml version="1.0" encoding="utf-8"?>
    <network-security-config>
    </network-security-config>
    ```
````

File: tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0250.md [hunk 7]
```diff
@@ -68,22 +68,13 @@ The output should contain:
...
-
-**Pass:**
-
-The test passes if any of the following are true:
-
-- `setJavaScriptEnabled` is explicitly set to `false` or _not used at all_ (inheriting the default value, `false`).
-- `setAllowContentAccess` method is explicitly set to `false`.
-- `setAllowUniversalAccessFromFileURLs` method is explicitly set to `false`.
+!!! note
+ The `setAllowContentAccess` method being set to `true` does not represent a security vulnerability by itself, but it can be used in combination with other vulnerabilities to escalate the impact of an attack.
```
Comment (cpholguera) [inline tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0250.md]:
```suggestion
    The `setAllowContentAccess` method being set to `true` does not represent a security vulnerability by itself, but it can be used in combination with other vulnerabilities to escalate the impact of an attack.
```

File: tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0251.md [hunk 8]
```diff
@@ -27,22 +27,13 @@ The output should contain a list of WebView instances and corresponding settings
...
-
-**Pass:**
-
-The test passes if any of the following are true:
-
-- `JavaScriptEnabled` is `false`.
-- `AllowContentAccess` is `false`.
-- `AllowUniversalAccessFromFileURLs` is `false`.
+!!! note
+ `AllowContentAccess` being `true` does not represent a security vulnerability by itself, but it can be used in combination with other vulnerabilities to escalate the impact of an attack.
```
Comment (cpholguera) [inline tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0251.md]:
```suggestion
    `AllowContentAccess` being `true` does not represent a security vulnerability by itself, but it can be used in combination with other vulnerabilities to escalate the impact of an attack.
```

File: tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0250.md [hunk 9]
```diff
@@ -68,22 +68,13 @@ The output should contain:
 
 ## Evaluation
 
-**Fail:**
-
-The test fails if all of the following are true:
+The test case fails if all of the following are true:
```
Comment (cpholguera) [inline tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0250.md]:
```suggestion
The test case fails if all of the following applies:
```

File: tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0251.md [hunk 10]
```diff
@@ -27,22 +27,13 @@ The output should contain a list of WebView instances and corresponding settings
 
 ## Evaluation
 
-**Fail:**
-
-The test fails if all of the following are true:
+The test case fails if all of the following are true:
```
Comment (cpholguera) [inline tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0251.md]:
```suggestion
The test case fails if all of the following applies:
```

File: tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0252.md [hunk 11]
```diff
@@ -68,20 +68,8 @@ The output should contain a list of WebView instances where the abovementioned m
 
 ## Evaluation
 
-The evaluation of this test is based on the [API behavior across different Android versions](../../../Document/0x05h-Testing-Platform-Interaction.md#webview-local-file-access-settings).
-
-**Fail:**
-
-The test fails if:
+The test case fails if (based on the [API behavior across different Android versions](../../../Document/0x05h-Testing-Platform-Interaction.md#webview-local-file-access-settings)):
```
Comment (cpholguera) [inline tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0252.md]:
```suggestion
The test case fails if all of the following applies (based on the [API behavior across different Android versions](../../../Document/0x05h-Testing-Platform-Interaction.md#webview-local-file-access-settings)):
```

File: tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0253.md [hunk 12]
```diff
@@ -27,20 +27,11 @@ The output should contain a list of WebView instances and corresponding settings
 
 ## Evaluation
 
-**Fail:**
-
-The test fails if all of the following are true:
+The test case fails if all of the following are true:
```
Comment (cpholguera) [inline tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0253.md]:
```suggestion
The test case fails if all of the following applies:
```

File: tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0253.md [hunk 13]
```diff
@@ -27,20 +27,11 @@ The output should contain a list of WebView instances and corresponding settings
...
-
-**Pass:**
-
-The test passes if any of the following are true:
-
-- `AllowFileAccess` is `false`.
-- `AllowFileAccessFromFileURLs` is `false`.
-- `AllowUniversalAccessFromFileURLs` is `false`.
+!!! note
+ `AllowFileAccess` being `true` does not represent a security vulnerability by itself, but it can be used in combination with other vulnerabilities to escalate the impact of an attack.
```
Comment (cpholguera) [inline tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0253.md]:
```suggestion
    `AllowFileAccess` being `true` does not represent a security vulnerability by itself, but it can be used in combination with other vulnerabilities to escalate the impact of an attack.
```

File: tests-beta/ios/MASVS-AUTH/MASTG-TEST-0268.md [hunk 14]
```diff
@@ -27,8 +27,7 @@ The output should contain a list of locations where relevant APIs are used.
...
-
-**Note:** Using `kSecAccessControlUserPresence` or `kSecAccessControlDevicePasscode` is not inherently a vulnerability, but in high-security applications (e.g., finance, government, health), their use can represent a weakness or misconfiguration that reduces the intended security posture. This issue is therefore better categorized as a security weakness or hardening issue, not a critical vulnerability.
+!!! note
+ Using `kSecAccessControlUserPresence` or `kSecAccessControlDevicePasscode` is not inherently a vulnerability, but in high-security applications (e.g., finance, government, health), their use can represent a weakness or misconfiguration that reduces the intended security posture. This issue is therefore better categorized as a security weakness or hardening issue, not a critical vulnerability.
```
Comment (cpholguera) [inline tests-beta/ios/MASVS-AUTH/MASTG-TEST-0268.md]:
```suggestion
    Using `kSecAccessControlUserPresence` or `kSecAccessControlDevicePasscode` is not inherently a vulnerability, but in high-security applications (e.g., finance, government, health), their use can represent a weakness or misconfiguration that reduces the intended security posture. This issue is therefore better categorized as a security weakness or hardening issue, not a critical vulnerability.
```

File: tests-beta/ios/MASVS-CODE/MASTG-TEST-0229.md [hunk 15]
```diff
@@ -32,13 +32,14 @@ The output should contain a list of symbols of the main binary and each shared l
...
+The test case fails if any binary or library is not purely Swift but does not contain methods indicating stack canaries like `objc_autorelease` or `objc_retainAutorelease`.
 
-**Note:** Checking for the `__stack_chk_fail` symbol only indicates that stack smashing protection is enabled somewhere in the app. While stack canaries are typically enabled or disabled for the entire binary, there may be corner cases where only parts of the application are protected. For example, if the app developer statically links a library with stack smashing protection enabled, but disables it for the entire application.
+!!! note
+ Checking for the `__stack_chk_fail` symbol only indicates that stack smashing protection is enabled somewhere in the app. While stack canaries are typically enabled or disabled for the entire binary, there may be corner cases where only parts of the application are protected. For example, if the app developer statically links a library with stack smashing protection enabled, but disables it for the entire application.
```
Comment (cpholguera) [inline tests-beta/ios/MASVS-CODE/MASTG-TEST-0229.md]:
```suggestion
    Checking for the `__stack_chk_fail` symbol only indicates that stack smashing protection is enabled somewhere in the app. While stack canaries are typically enabled or disabled for the entire binary, there may be corner cases where only parts of the application are protected. For example, if the app developer statically links a library with stack smashing protection enabled, but disables it for the entire application.
```

File: tests-beta/ios/MASVS-CODE/MASTG-TEST-0230.md [hunk 16]
```diff
@@ -30,8 +30,9 @@ The output should contain a list of symbols of the main binary and each shared l
...
+The test case fails if any binary or library containing Objective-C or Swift code is missing ARC-related symbols. The presence of symbols such as `_objc_msgSend` (Objective-C) or `_swift_allocObject` (Swift) without corresponding ARC symbols indicates that ARC may not be enabled.
 
-**Note:** Checking for these symbols only indicates that ARC is enabled somewhere in the app. While ARC is typically enabled or disabled for the entire binary, there can be corner cases where only parts of the application or libraries are protected. For example, if the app developer statically links a library that has ARC enabled, but disables it for the entire application.
+!!! note
+ Checking for these symbols only indicates that ARC is enabled somewhere in the app. While ARC is typically enabled or disabled for the entire binary, there can be corner cases where only parts of the application or libraries are protected. For example, if the app developer statically links a library that has ARC enabled, but disables it for the entire application.
```
Comment (cpholguera) [inline tests-beta/ios/MASVS-CODE/MASTG-TEST-0230.md]:
```suggestion
    Checking for these symbols only indicates that ARC is enabled somewhere in the app. While ARC is typically enabled or disabled for the entire binary, there can be corner cases where only parts of the application or libraries are protected. For example, if the app developer statically links a library that has ARC enabled, but disables it for the entire application.
```

File: tests-beta/ios/MASVS-CODE/MASTG-TEST-0230.md [hunk 17]
```diff
@@ -30,8 +30,9 @@ The output should contain a list of symbols of the main binary and each shared l
...
 
 If you want to be sure that specific security-critical methods are adequately protected, you need to reverse-engineer each of them and manually check for ARC, or request the source code from the developer.
```
Comment (cpholguera) [inline tests-beta/ios/MASVS-CODE/MASTG-TEST-0230.md]:
```suggestion
    
    If you want to be sure that specific security-critical methods are adequately protected, you need to reverse-engineer each of them and manually check for ARC, or request the source code from the developer.
```

File: tests-beta/ios/MASVS-STORAGE/MASTG-TEST-0313.md [hunk 18]
```diff
@@ -38,12 +38,11 @@ The output should contain a list of locations where the app:
...
 
 - `isSecureTextEntry` is not enabled, or
 - `autocorrectionType` is set to `default` or `yes`, or
 - `spellCheckingType` is set to `default` or `yes`.
 
-The test passes if all security-sensitive inputs are protected against keyboard caching.
-
-**Note:** Depending on the app's threat model, some text fields may not require disabling spell checking. However, since enabling `isSecureTextEntry` implicitly disables both autocorrection and spell checking, and since explicit guarantees are otherwise limited, it is generally advisable to disable all three attributes for any text field that may handle sensitive information.
+!!! note
+ Depending on the app's threat model, some text fields may not require disabling spell checking. However, since enabling `isSecureTextEntry` implicitly disables both autocorrection and spell checking, and since explicit guarantees are otherwise limited, it is generally advisable to disable all three attributes for any text field that may handle sensitive information.
```
Comment (cpholguera) [inline tests-beta/ios/MASVS-STORAGE/MASTG-TEST-0313.md]:
```suggestion
    Depending on the app's threat model, some text fields may not require disabling spell checking. However, since enabling `isSecureTextEntry` implicitly disables both autocorrection and spell checking, and since explicit guarantees are otherwise limited, it is generally advisable to disable all three attributes for any text field that may handle sensitive information.
```

File: tests-beta/ios/MASVS-STORAGE/MASTG-TEST-0314.md [hunk 19]
```diff
@@ -31,12 +31,11 @@ The output should allow the tester to associate each text entry with the corresp
...
 
 - `isSecureTextEntry` is not enabled, or
 - `autocorrectionType` is set to `default` or `yes`, or
 - `spellCheckingType` is set to `default` or `yes`.
 
-The test passes if all security-sensitive inputs are protected against keyboard caching.
-
-**Note:** This test may produce false negatives if the app uses custom text input controls that do not rely on standard UIKit classes such as `UITextField` or `UITextView` (for example in custom UI frameworks or game engines), or if text entry is handled through nonstandard abstractions that prevent reliable observation of input traits at runtime.
+!!! note
+ This test may produce false negatives if the app uses custom text input controls that do not rely on standard UIKit classes such as `UITextField` or `UITextView` (for example in custom UI frameworks or game engines), or if text entry is handled through nonstandard abstractions that prevent reliable observation of input traits at runtime.
```
Comment (cpholguera) [inline tests-beta/ios/MASVS-STORAGE/MASTG-TEST-0314.md]:
```suggestion
    This test may produce false negatives if the app uses custom text input controls that do not rely on standard UIKit classes such as `UITextField` or `UITextView` (for example in custom UI frameworks or game engines), or if text entry is handled through nonstandard abstractions that prevent reliable observation of input traits at runtime.
```

**Review by cpholguera — APPROVED** [review]
